tests/test_texmobject.py

```diff
@@ -15,13 +15,13 @@ def test_SingleStringMathTex():
     assert Path(config.media_dir, "Tex", "79822967f1fa1935.svg").exists()
 
 
-@pytest.mark.parametrize(
+@pytest.mark.parametrize(  # : PT006
```

Comment:
what does this comment mean?

Reply:
That's supposed to be a noqa. Flake had a weird error here.